### PR TITLE
Make it rich

### PR DESCRIPTION
**Pull request by asvetlov**

### CODE REVIEW

**Comment by asvetlov** [comment]
```
$ neuro config show
User Configuration:                               
 User Name            asvetlov                    
 Current Cluster      default                     
 API URL              https://dev.neu.ro/api/v1   
 Docker Registry URL  https://registry-dev.neu.ro 
Resource Presets:                                                            
 Name        #CPU   Memory   Preemptible   GPU                    Jobs Avail 
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 cpu-micro    0.1   100.0M        ×                                      190 
 cpu-small      1     4.0G        ×                                       17 
 cpu-large      3    12.0G        ×                                        2 
 gpu-small      5    48.0G        ×        1 x nvidia-tesla-k80            3 
```
plus bold/italic

File: tests/cli/conftest.py [hunk 1]
```diff
@@ -123,3 +126,204 @@ def _run_cli(arguments: List[str]) -> SysCapWithCode:
...
+            ]
+        # "right" is the expected base against which we compare "left",
+        # see https://github.com/pytest-dev/pytest/issues/3333
+        explanation += [
+            line.strip("\n")
+            for line in ndiff(right.splitlines(keepends), left.splitlines(keepends))
+        ]
+        explanation.append("")
+        explanation.append(f"'cat {self.rel(lft.path)}' to see the test output.")
+        explanation.append(f"'cat {self.rel(rgt.path)}' to see the reference.")
```
Comment (asvetlov) [inline tests/cli/conftest.py:271]: On failure, pytest displays instructions on how to see the expected and actual texts.
Both are stored in files, the expected version is saved by git.

`pytest --rich-gen` updates all *expected* files (references).
You can do it yourself but automation is very handy.
The changes are pretty visible by PR diffs.

File: tests/cli/formatters/ascii/TestAliasesFormatter.test_output.ref [hunk 2]
```diff
@@ -0,0 +1,6 @@
+           ╷                              
+ [1m [0m[1mAlias   [0m[1m [0m│[1m [0m[1mDescription                [0m[1m [0m
```
Comment (asvetlov) [inline tests/cli/formatters/ascii/TestAliasesFormatter.test_output.ref:2]: An example of an expected reference file.
ANSI symbols are saved, `cat <file>` shows how it looks on the terminal.
Usually, there is no need to update the file manually but run `pytest --rich-gen`

File: tests/cli/formatters/test_config_formatters.py [hunk 3]
```diff
@@ -220,62 +152,43 @@ def test_format_time_9999h_59m(self) -> None:
...
             total_gpu_run_time_minutes=9923, total_non_gpu_run_time_minutes=None
         )
         out = QuotaFormatter()(quota)
-        assert out == "\n".join(
-            [
-                f"{bold_start}GPU:{bold_end} 9923m",
-                f"{bold_start}CPU:{bold_end} infinity",
-            ]
-        )
+        rich_cmp(out)
```
Comment (asvetlov) [inline tests/cli/formatters/test_config_formatters.py:175]: An assertion example.

File is auto-calculated from test module, test class, and test function.

**Comment by romasku** [comment]
Some E2E test are also failing because of formatting issues: `test_print_config`, `test_get_cluster_users`.
Also `test_images_delete` looks like broken on master (this test fails when I try to run locally), so you can ignore it.

**Pull request merged**